RxJavaTemplate/buildSrc/src/main/java/Dependencies.kt

```diff
@@ -0,0 +1,127 @@
+object AppModule {
+    val implementations = listOf(
+        KOTLIN_STDLIB_JDK7,
+        KOTLIN_STDLIB,
+
+        ANDROID_MATERIAL,
+        ANDROIDX_CORE,
+        ANDROIDX_FRAGMENT_KTX,
+        ANDROIDX_LEGACY_SUPPORT,
+        ANDROIDX_APPCOMPAT,
+        ANDROIDX_RECYCLERVIEW,
+        ANDROIDX_CONSTRAINT_LAYOUT,
+        ANDROIDX_MULTIDEX,
+
+        ANDROIDX_NAVIGATION_RUNTIME_KTX,
+        ANDROIDX_NAVIGATION_FRAGMENT_KTX,
+        ANDROIDX_NAVIGATION_UI_KTX,
+
+        RX_BINDING,
+        RX_BINDING_CORE,
```

Comment:
How about putting these two below `ANDROIDX_LIFECYCLE_RUNTIME,`? Keeping the `ANDROIDX` dependencies close to each other would make it prettier. 😃